### ISSUE TITLE
Swift Compiler Error in TapNibInputView: '@objc' property requires iOS 13.0

### ISSUE THREAD

**Issue by mirland** [Tap-Payments/goSellSDK-iOS#74]
Hi, I have exactly [this](https://github.com/Tap-Payments/gosellSDK-Flutter/issues/67) issue with the iOS native dependency. That issue was happening in Flutter, but I'm facing it on the native dependency, too.

```
something/Pods/TapResponderChainInputViewV2/TapResponderChainInputViewV2/TapResponderChainInputViewV2/Source/TapResponderChainInputView.swift:198:16 '@objc' property in extension of subclass of 'TapNibInputView' requires iOS 13.0.0
```

Xcode Version: 16.1 (16B40)
Mac: Apple M3 Pro
Pod version: 2.3.42 (latest)
minimum ios version: 14

<img width="535" alt="image" src="https://github.com/user-attachments/assets/aaeab6c3-e35b-42ca-bfd0-833d3d5f9c4d">

**Comment by mAllamTapPayments** [comment]
@mirland 
Please Could you search for deployment in xCode send me the min deployment version like this 
![Screenshot 2024-11-26 at 6 31 18 PM](https://github.com/user-attachments/assets/c14f9718-077c-46f2-838c-61563191ad29)

**Comment by mirland** [comment]
Hi, it's using 14.0, so the version is ok. However, today I could fix the issue by updating cocoapods from  1.16.1 to  1.16.2. I don't know what the problem was, but I couldn't use TAP with that version. Now it's working. Thanks
